packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.js

```diff
@@ -5,15 +5,24 @@ import {LightningElement, api} from 'lwc';
  * This component should be used inside a container with a CSS position attribute set to the value `relative`
  * @category Internal
  * @example
- * <c-quantic-tooltip value={value}></c-quantic-tooltip>
+ * <c-quantic-tooltip light-theme>
+ *  <div slot="content">Tooltip content</div>
+ * </c-quantic-tooltip>
  */
 export default class QuanticTooltip extends LightningElement {
   /**
    * @api
    * @type {string}
+   * @deprecated
```

Comment:
I accidentally exposed an unused property 😔 will create a ticket to  remove it in the V3.

Reply:
You are right I just confirmed that this shouldn't break anything 👌